llvm/test/CodeGen/RISCV/inline-asm-mem-constraint-2.ll

```diff
@@ -0,0 +1,96 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -verify-machineinstrs -no-integrated-as < %s \
+; RUN:   | FileCheck -check-prefixes=RV32I %s
+; RUN: llc -mtriple=riscv64 -verify-machineinstrs -no-integrated-as < %s \
+; RUN:   | FileCheck -check-prefixes=RV64I %s
+; RUN: llc -mtriple=riscv32 -code-model=medium -verify-machineinstrs -no-integrated-as < %s \
+; RUN:   | FileCheck -check-prefixes=RV32I-MEDIUM %s
+; RUN: llc -mtriple=riscv64 -code-model=medium -verify-machineinstrs -no-integrated-as < %s \
+; RUN:   | FileCheck -check-prefixes=RV64I-MEDIUM %s
+;
+; integrated-as fails with error: unexpected token
+;   sw zero, %lo(eg)(a0) \n sw zero, %lo(eg)(a0)
```

Comment:
Use \0A instead of \n?

Reply:
That was in the test before my chagne. I just split inline-asm-mem-contriant.ll into two tests: the one that fails with intergrated-as and the second one that works correctly.

I mean I can change it, not a problem. But originally it was added with '\n'.